forest/benchmarking/tomography.py

```diff
@@ -163,6 +167,98 @@ def linear_inv_state_estimate(results: List[ExperimentResult],
     rho = pinv(measurement_matrix) @ expectations
     return unvec(rho)
 
+def compressed_sensing_state_estimate(results: List[ExperimentResult],
+                              qubits: List[int]) -> np.ndarray:
+    """
+    Estimate a quantum state using compressed sensing
```

Comment:
Estimate a quantum state using compressed sensing, i.e. by constrained trace minimization (a.k.a. the matrix Dantzig selector).